### DISCUSSION TITLE
Thank you Alex and the Immich team

### DISCUSSION THREAD

**Discussion by Eirikr70** [immich-app/immich#5791] [General]
I just wanted to congratulate the team behind Immich. The job you make is impressive ! Recently you have considerably optimised the server side, which my raspberry Pi4 can handle along with a dozen other services. And the service offered is great ! I have never used GPhotos, but Immich wins the competition with any other app I have used.
Thanks again !

**Comment by arpit4** [comment]
Likewise, thanks a lot team,
Immich is a dream come true.
